rl_studio/agents/f1/train_ddpg.py

```diff
@@ -0,0 +1,269 @@
+from collections import deque
+import time
+import random
+import os
+import time
+from tqdm import tqdm
+from cprint import cprint
+import numpy as np
+import random
+import utils
+from icecream import ic
+from datetime import datetime, timedelta
+import numpy as np
+import gym
+import pandas as pd
+from algorithms.ddpg import ModifiedTensorBoard, OUActionNoise, Buffer, DDPGAgent
+from envs.gazebo_env import *
+
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow.keras import Input, Model, layers
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import Dense, Dropout, Conv2D, MaxPooling2D, Activation, Flatten, BatchNormalization, Lambda
```

Comment:
Same here. You are mixing the "agent" and the "algorithm". We must separate the code.

Reply:
I believe this file follows your current structure.
You have:
/agents/f1/train_qlearn.py
and
/algorithms/qlearn.py
So, when you want to train a F1 agent, the train_qlearn.py is calling the appropriate algorithm (learn).

Thus, now, 
/agents/f1/train_ddpg.py which is calling 
/algorithms/ddp.py